### ISSUE TITLE
Modifications in Message Bar

### ISSUE THREAD

**Issue by SamarthKadam** [SamarthKadam/ChatBox#300]
![Screenshot (630)](https://github.com/SamarthKadam/ChatBox/assets/87573175/3393a3ba-f2c3-4830-aebb-6d8b6ef51757)
We can see that timestamp doesn't appear in a same row. It needs to be in a line

**Comment by BHS-Harish** [comment]
Hey @SamarthKadam 

Please could you assign the issue to me....

**Comment by SamarthKadam** [comment]
Yes @BHS-Harish You can go ahead and Please try to complete it asap
